x-pack/plugin/security/cli/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'elasticsearch.build'
+
+archivesBaseName = 'elasticsearch-security-cli'
+dependencies {
+    compileOnly "org.elasticsearch:elasticsearch:${version}"
+    compile 'org.bouncycastle:bcprov-jdk15on:1.59'
+    compile 'org.bouncycastle:bcpkix-jdk15on:1.59'
+}
+
+jar {
+    from(project(':x-pack:plugin:core').sourceSets.main.output.classesDirs) {
+        include 'org/elasticsearch/xpack/core/ssl/CertificateTool.class'
+        include 'org/elasticsearch/xpack/core/ssl/CertificateGenerateTool.class'
+        include 'org/elasticsearch/xpack/core/ssl/CertGenUtils.class'
+    }
+}
+
+dependencyLicenses {
+    mapping from: /bc.*/, to: 'bouncycastle'
+}
+
+licenseHeaders.enabled = false
```

Comment:
Can you explain what is happening here?

Reply:
I can certainly try.  I started off without the `licenseHeaders.enabled = false` and precommit checks failed with: 

```
> Task :x-pack:plugin:security:cli:licenseHeaders FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':x-pack:plugin:security:cli:licenseHeaders'.
> You must specify at least one file to create the report for.
```

I interpreted this to be an error caused by the licenseHeaders task looking for Java files in order to check the license heades in security-cli and failing because there aren't any. Depending on whether I'll bring in the source files (https://github.com/elastic/elasticsearch/pull/32193#discussion_r203787096) , I'll remove this but what should have been the correct approach in this case?

Reply:
Sorry, I was unclear. I meant can you leave a comment in the build.gradle about this. Of course, this changes if we do bring in the source files.